src/CommunityToolkit/Xamarin.CommunityToolkit/Views/Popup/BasePopup.shared.cs

```diff
@@ -18,7 +18,7 @@ public abstract class BasePopup : VisualElement
 		/// </summary>
 		protected BasePopup()
 		{
-			Color = Color.White;
+			Color = Color.Default;
```

Comment:
Actually, all of that can be removed, because we have the bindable default values.